.idea/runConfigurations/Run_prod.xml

```diff
@@ -7,7 +7,7 @@
         <option name="command" value="" />
         <option name="containerName" value="todo-app-prod" />
         <option name="entrypoint" value="" />
-        <option name="commandLineOptions" value="-p 5000:5000 --env-file .env" />
+        <option name="commandLineOptions" value="-p 5000:5000 --env-file .env --env PORT=5000" />
```

Comment:
Well remembered to change here, we should also update the Readme command for running the prod container